bigquery_etl/routine/parse_routine.py

```diff
@@ -14,40 +14,49 @@
 import yaml
 
 from bigquery_etl.metadata.parse_metadata import METADATA_FILE
-from bigquery_etl.util.common import project_dirs
 
 
-UDF_DIRS = ("udf", "udf_js")
-MOZFUN_DIR = ("sql/mozfun",)
 UDF_CHAR = "[a-zA-z0-9_]"
 UDF_FILE = "udf.sql"
```

Comment:
`s/(UDF_FILE, PROCEDURE_FILE)/ROUTINE_FILE` - they are used together in almost every instance now, no?